### ISSUE TITLE
Any updates on the English version book? 

### ISSUE THREAD

**Issue by DigitalGabriele** [gaoxiang12/slambook-en#11]

**Comment by gaoxiang12** [comment]
Yes, I'll start to do the translation after finishing the second Mandarin version.

**Comment by DigitalGabriele** [comment]
Any hints on when that would happen? I would definitely donate/buy the English version book.

**Comment by SKioresku** [comment]
I would buy the English version book too. Can You add pages with IMU sensors to the book please?

**Comment by gaoxiang12** [comment]
I think would be mid 2019 if everything is fine. English version will probably be a public version. I'll put the pdf in github too.

**Comment by srnetsr** [comment]
Thank you for your publication/translation.
I am look forward to reading your book soon.
I hope uploading the book of English version will not be delayed...

**Comment by srnetsr** [comment]
Hi Dr.Gao (@gaoxiang12),
I thank you for your publication/translation.
It would be very thankful if you upload chapters which have been translated already.
(ex. chapter 3 and 4....before uploading the whole book chapters...)
I cannot wait until you upload the whole book... XD

**Comment by DigitalGabriele** [comment]
I am definitely on the side with @srnetsr. It would be great if you, @gaoxiang12, could upload any chapters you have translated.

**Comment by Roshnee** [comment]
Hello,

Any updates on the english version yet? I don't find it after chapter 2. Please do release an english version and it would be really really great if it done as soon as possible. It would be of great help.

Thanks in advance

**Comment by gaoxiang12** [comment]
Hi @Roshnee ,
Sorry for the late reply because I was doing some publish work recently (for example I have to design the cover and ask some professors to write a recommendation). I'm starting to the translation now, and the English version is switched to Latex. You can download the compiled pdf [here](https://github.com/gaoxiang12/slambook-en/blob/master/slambook-en.pdf) and also visit the latex source code in this repo.

**Comment by DigitalGabriele** [comment]
@gaoxiang12, if you need some volunteers to help you with the book, I would definitely be interested.

**Comment by Napester** [comment]
Hi Dr @gaoxiang12 
any updates regarding the remaining chapters of the English version?

**Comment by sumitsarkar1** [comment]
Hi Dr. @gaoxiang12 any updates on the english translation of the chapters 5 onwards. Eagerly waiting for the rest of the book chapters

**Comment by yuthikasagarage** [comment]
Dr. @gaoxiang12 any updates on the english translation ?

**Comment by rajy4683** [comment]
@gaoxiang12 Thanks for the wonderful chapters so far. Any update when the rest of chapters (Chapter 7 onwards) will be available.
